draft-packages/button/KaizenDraft/Button/components/GenericButton.tsx

```diff
@@ -52,6 +53,9 @@ export type LabelProps = {
   primary?: boolean
   secondary?: boolean
   reverseColor?: "cluny" | "peach" | "seedling" | "wisteria" | "yuzu"
+  working?: boolean | null
```

Comment:
Any particular reason for using `null` over allowing the prop to be undefined when not specified? 

```
props.working -> undefined or false -> don't show working state
props.working -> true -> show working state
```

Reply:
Yes good point